java/tools/src/test/java/org/apache/arrow/tools/TestFileRoundtrip.java

```diff
@@ -78,19 +77,17 @@ public void testDiffFolder() throws Exception {
 
   @Test
   public void testNotPreparedInput() throws Exception {
-    File testInFile = testFolder.newFile("testIn.arrow");
-    File testOutFile = testFolder.newFile("testOut.arrow");
+    File testInFile = new File(testFolder, "testIn.arrow");
+    File testOutFile = new File(testFolder, "testOut.arrow");
 
     String[] args = {"-i", testInFile.getAbsolutePath(), "-o", testOutFile.getAbsolutePath()};
-
-    // In JUnit 5, since the file itself is not created, the exception and message will be
-    // different.
     Exception exception =
         assertThrows(
-            InvalidArrowFileException.class,
+            IllegalArgumentException.class,
             () -> {
               new FileRoundtrip(System.err).run(args);
             });
-    assertEquals("file too small: 0", exception.getMessage());
+
+    assertTrue(exception.getMessage().contains("input file not found"));
```

Comment:
Is it possible to keep testing the "old" path as well? This would mean, create the output file (as an empty file) and ensure we throw "file to small" error